front/lib/api/tables.ts

```diff
@@ -283,20 +284,33 @@ async function rowsFromCsv(
     const record = anyRecord as string[];
 
     if (!header) {
-      header = record;
-      continue;
-    }
-
-    header = header.map((h) => h.trim().toLowerCase());
-    const headerSet = new Set<string>();
-    for (const h of header) {
-      if (headerSet.has(h)) {
-        return new Err({
-          type: "duplicate_header",
-          message: `Duplicate header: ${h}.`,
-        });
+      header = [];
+      const firstRecordCells = record.map((h) => h.trim().toLocaleLowerCase());
+      let emptyHeaderCellFound = false;
+      for (const r of firstRecordCells) {
```

Comment:
`nit` A tiny suggestion to make this a bit easier on the eyes 🙃:

```
const firstEmptyCellIndex = firstRecordCells.findIndex(cell => cell === "");

if (firstEmptyCellIndex !== -1) {
 // Check for non-empty cells after the index of the first empty cell.
}
```

Reply:
Fair ! updating